src/core/annotation.js

```diff
@@ -3800,6 +3805,18 @@ class InkAnnotation extends MarkupAnnotation {
       appearanceStreamDict.set("Matrix", matrix);
     }
 
+    if (opacity !== 1) {
+      const resources = new Dict(xref);
+      const extGState = new Dict(xref);
+      const r0 = new Dict(xref);
+      r0.set("AIS", false);
```

Comment:
Is this property actually necessary here?

The reason that I ask is that in the PDF.js library [we don't actually support that property](https://github.com/mozilla/pdf.js/blob/8ebd2d3dfd0c32086314014f8006c91913c2f746/src/core/evaluator.js#L1138-L1142), and there's even an old issue related to it (see #6967). Hence it seems slightly strange, at least to me, that we're generating data that we're actually not able parse/use ourselves.